frontend/src/components/FormInput/index.tsx

```diff
@@ -9,14 +10,15 @@ import { GridItem } from "../GDXForm/FormLayout/GridItem";
 import { ReadField } from "components/ReadForm/ReadField";
 import { IOption, IFormInput } from "../../types";
 import moment from "moment";
+import dayjs from "dayjs";
 
 export const FormInput = ({
   setFieldValue,
   fieldValue,
   fieldName,
   fieldType,
   fieldLabel,
-  handleChange,
+  handleChange = () => {},
```

Comment:
Just confirming you meant to do this, i'm guessing because you want an initial value.

Reply:
Yes it's needs a default.